src/Nethereum.RPC/Eth/EthGetCode.cs

```diff
@@ -49,12 +49,17 @@ public EthGetCode(IClient client) : base(client, ApiMethods.eth_getCode.ToString
             return base.SendRequestAsync(id, cancellationToken, address.EnsureHexPrefix(), block);
         }
 
-        public Task<string> SendRequestAsync(string address,
+        public async Task<string> SendRequestAsync(string address,
                                              object id = null,
                                              CancellationToken cancellationToken = default(CancellationToken))
         {
             if (address == null) throw new ArgumentNullException(nameof(address));
-            return base.SendRequestAsync(id, cancellationToken, address.EnsureHexPrefix(), DefaultBlock);
+            var res = await this.Client.SendRequestAsync<object>(this.BuildRequest(address.EnsureHexPrefix(), DefaultBlock, id));
+            // Some servers may return empty array for non-contract addresses instead of "0x", so check for array
+            if (res is Newtonsoft.Json.Linq.JArray)
```

Comment:
@webwarrior-ws you also need to check if the array is empty

Reply:
What exception to throw when it is not?

Reply:
I don't know, UnexpectedReplyException? and then put the json in the message

Reply:
JSON is unavailable at this location, only result value

Reply:
Do we have `UnexpectedReplyException` somewhere or are you suggesting to define new type?

Reply:
I'm suggesting a new type unless you can find an adequate one

Reply:
> JSON is unavailable at this location, only result value

The result value is an object representation of the JSON result, I'm sure it can be converted back to the original string somehow.